app/views/orders/index.html.erb

```diff
@@ -0,0 +1,94 @@
+<div class="container justify-content-center mt-5 text-center w-50">
+  <div class="row">
+    <%= form_with url: filter_orders_path, method: :get, class: "form-inline col-6" do |form| %> 
+      <div class="input-group mb-3"> 
+        <%= form.text_field :quantity, class: 'form-control mr-sm-2', placeholder: 'Filter orders by Quantity'%> 
+        <%= form.submit "Filter", class: 'btn btn-outline-success my-2 my-sm-0' %>
+      </div> 
+    <% end %>
+    <%= form_with url: search_orders_path, method: :get, class: "form-inline col-6" do |form| %> 
+      <div class="input-group mb-3"> 
+        <%= form.text_field :title, class: 'form-control mr-sm-2', placeholder: "Which product's order would you like to Search"%> 
+        <%= form.submit "Search", class: 'btn btn-outline-success my-2 my-sm-0' %>
+      </div> 
+    <% end %>
+  </div>
+
+  <div style="height:400px; overflow:auto;">
+    <% if defined? @orders_by_product_name %>
```

Comment:
We shouldn't check this here, instead we should pass blank array if data not present.